cmd/commands/runtime.go

```diff
@@ -1797,7 +1872,16 @@ func configureAppProxy(ctx context.Context, opts *RuntimeInstallOptions, rt *run
 					ServicePort: store.Get().AppProxyServicePort,
 				},
 			},
-		})
+		}
+
+		if opts.IngressControllerType == NginxEnterprise {
```

Comment:
I don't want those lines to repeat >3 times, need to move to a function.

Reply:
they are not repeating. the logic is different in the 3 cases